osu.Game.Rulesets.Taiko.Tests/Mods/TestSceneTaikoModFlashlight.cs

```diff
@@ -16,5 +20,37 @@ public class TestSceneTaikoModFlashlight : TaikoModTestScene
 
         [Test]
         public void TestComboBasedSize([Values] bool comboBasedSize) => CreateModTest(new ModTestData { Mod = new TaikoModFlashlight { ComboBasedSize = { Value = comboBasedSize } }, PassCondition = () => true });
+
+        [Test]
+        public void TestFlashlightAlwaysHasNonZeroSize()
+        {
+            bool failed = false;
+
+            CreateModTest(new ModTestData
+            {
+                Mod = new TestTaikoModFlashlight { ComboBasedSize = { Value = true } },
+                Autoplay = false,
+                PassCondition = () =>
+                {
+                    failed |= this.ChildrenOfType<TestTaikoModFlashlight.TestTaikoFlashlight>().SingleOrDefault()?.FlashlightSize.Y == 0;
+                    return !failed;
```

Comment:
Writing this the obvious way causes a false pass, because the test beatmap has a break, and the break path will correctly set the flashlight size, making the test not fail.